clustering/common/src/main/java/org/jboss/as/clustering/controller/ReloadRequiredAddStepHandler.java

```diff
@@ -22,8 +22,6 @@
 
```

Comment:
Why not just remove this entire class as it appears to be exactly the same as what's in the core controller?